nlu/pipe/col_substitution/col_substitution_OS.py

```diff
@@ -1085,6 +1085,31 @@ def substitute_summarizer_cols(c, cols, nlu_identifier=True):
             # new_cols[col]= f"{new_base_name}_confidence"
     return new_cols
 
+def substitute_text_generator_cols(c, cols, nlu_identifier=True):
+    """
+    rename cols with base name either <t5> or if not unique <t5_<task>>
+    """
+    new_cols = {}
+    new_base_name = 'text generated' if nlu_identifier == 'UNIQUE' else f'text generated_{nlu_identifier}'
+    for col in cols:
```

Comment:
no white spaces for column names. should be text_generated. You can also re-use the one from gpt2 I think, no need to create this